tools/dungeonjson/dungeonjson.cpp

```diff
@@ -719,6 +720,42 @@ string generate_move_data_text(Json data) {
     return text.str();
 }
 
+string get_compressed_move_id(string move_macro) {
+    ostringstream text;
+    text << "\n";
+    int move_id = get_move_id(move_macro);
```

Comment:
How does it handle moves that aren't in the list? Just thinking in case someone is trying to add a new move/learnset.

Reply:
Right now it returns a move ID of 0. If it's worth thinking ahead to new moves, I can switch the map access to use `.at()` so that it throws an error when a move is not found.

Reply:
Yeah, I think that might be better. If you can add a message saying to add it to `move_id_json.cpp`, that would be optimal.